### PR TITLE
docs: Add logo, update year in license, add get started guide

### PR DESCRIPTION
**Pull request by adityasaky**
~Draft until I also add the get started guide.~

~LMK if the logo in the readme is too much.~ Now at 25% width.

### CODE REVIEW

**Comment by adityasaky** [comment]
great, thanks!

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2023 gittuf a Series of LF Projects, LLC
```
Comment (wlynch) [inline LICENSE:1]: My understanding is we don't need to continuously bump this, and LF recommendation is that we don't need the year at all https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

Comment (adityasaky) [reply LICENSE:1]: Interesting. I'll take the year out altogether then. cc @hythloda

Comment (adityasaky) [reply LICENSE:1]: Done.

**Review by wlynch — APPROVED** [review]

**Pull request merged**